config/locales/en.yml

```diff
@@ -379,6 +379,12 @@ en:
         heading: "Delete this project list?"
         text: "This action will not delete any project the list contains. Are you sure you want to delete this project list?"
     settings:
+      header_details: Basic details
+      header_status: Project status
+      header_relations: Project relations
+      button_update_details: Update details
+      button_update_status: Update status
+      button_update_parent_project: Update parent project
```

Comment:
Not sure what convention we're using for I18n strings?

Should we sort keys alphabetically?